lib/src/main/java/com/fabernovel/statefullayout/StatefulLayout.kt

```diff
@@ -142,7 +142,7 @@ class StatefulLayout : FrameLayout, StateContainer<Int, State> {
     }
 
     override fun onViewAdded(child: View?) {
-        super.onViewAdded(child)
```

Comment:
removing the call to the super method : is it done on purpose ?

Reply:
No it's a mistake